bin/docker_run.sh

```diff
@@ -12,4 +12,9 @@ if [ $? -ne 0 ]; then
     schematic migrations/
 fi
 
+if [ "$BRAINTREE_MERCHANT_ID" ]; then
+    echo "Braintree merchant ID found in environment, running braintree_config."
+    python manage.py braintree_config
```

Comment:
are you sure this can be run repetitively? I got this when I ran it: `django.db.utils.IntegrityError: (1062, "Duplicate entry 'mozilla-concrete-brick' for key 'public_id'")`